clippy_lints/src/methods/filter_map.rs

```diff
@@ -58,15 +59,15 @@ fn lint_filter_some_map_unwrap(
     map_arg: &hir::Expr<'_>,
     target_span: Span,
     methods_span: Span,
-) {
+) -> bool {
```

Comment:
IMO it would be better to separate function into one that check and return `bool` and one that just emit the lint. I was confused that the function name `lint_filter_some_map_unwrap` return `bool`.

Reply:
Thanks, changed it to be an `is_` function